WindowsWrapper.hpp

```diff
@@ -691,7 +694,7 @@ namespace Windows {
 		CreateProcessW(nullptr, commandLine.data(), processAttributes, threadAttribtues,
 		               inheritHandles, creationFlags, environments, currentDirectory, startUpInfo, &processInformation)
 		        >> checkWin32Result("CreateProcessW", errorValue, false);
-		Handle{processInformation.hThread};
+		void(Handle{processInformation.hThread});

```

Comment:
If this did triggered a warning on MSVC, then I guess it's better to directly replace it with `CloseHandle()`?

Reply:
If I remembered it right, VS has also suggested the same thing on my other projects. Anyway I think `CloseHandle` would at least be prettier.